main.go

```diff
@@ -216,13 +216,8 @@ func main() {
 			log.Info("Starting lease controller to report status")
 			leaseUpdater := lease.NewLeaseUpdater(
 				generatedClient,
-				"policy-controller",
+				"governance-policy-framework",
 				operatorNs,
-				lease.CheckAddonPodFunc(generatedClient.CoreV1(), operatorNs, "app=policy-framework"),
```

Comment:
We still need this health check so that policy-framework addon is green only when all three pods are running